From a2cf8d62a9bd7860419c7c5e204a7f0f56f56966 Mon Sep 17 00:00:00 2001
From: Dharma Balasubiramani <dharma.b@microchip.com>
Date: Sat, 15 Oct 2022 03:15:28 +0530
Subject: [PATCH] dts: include optee dtsi

- Include sama5d2-optee.dtsi in at91-sama5d27_som1_ek.dts
- Include sama5d2-optee.dtsi in at91-sama5d27_wlsom1_ek.dts

Signed-off-by: Dharma Balasubiramani <dharma.b@microchip.com>
Signed-off-by: Thomas Perrot <thomas.perrot@bootlin.com>
---
 arch/arm/boot/dts/microchip/at91-sama5d27_som1_ek.dts   | 1 +
 arch/arm/boot/dts/microchip/at91-sama5d27_wlsom1_ek.dts | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/microchip/at91-sama5d27_som1_ek.dts b/arch/arm/boot/dts/microchip/at91-sama5d27_som1_ek.dts
index f3ffb8f01d8a..9c781dce3425 100644
--- a/arch/arm/boot/dts/microchip/at91-sama5d27_som1_ek.dts
+++ b/arch/arm/boot/dts/microchip/at91-sama5d27_som1_ek.dts
@@ -9,6 +9,7 @@
  */
 /dts-v1/;
 #include "at91-sama5d27_som1.dtsi"
+#include "sama5d2-optee.dtsi"
 #include <dt-bindings/mfd/atmel-flexcom.h>
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
diff --git a/arch/arm/boot/dts/microchip/at91-sama5d27_wlsom1_ek.dts b/arch/arm/boot/dts/microchip/at91-sama5d27_wlsom1_ek.dts
index 15239834d886..00782000373e 100644
--- a/arch/arm/boot/dts/microchip/at91-sama5d27_wlsom1_ek.dts
+++ b/arch/arm/boot/dts/microchip/at91-sama5d27_wlsom1_ek.dts
@@ -8,6 +8,7 @@
  */
 /dts-v1/;
 #include "at91-sama5d27_wlsom1.dtsi"
+#include "sama5d2-optee.dtsi"
 #include <dt-bindings/input/input.h>
 
 / {
@@ -265,4 +266,3 @@ &usb2 {
 &watchdog {
 	status = "okay";
 };
-
-- 
2.45.2

